src/Promitor.Core.Scraping/Configuration/Model/Metrics/MetricDefinition.cs

```diff
@@ -4,6 +4,26 @@ namespace Promitor.Core.Scraping.Configuration.Model.Metrics
 {
     public abstract class MetricDefinition
     {
+        protected MetricDefinition()
+        {
+        }
+
+        protected MetricDefinition(
```

Comment:
Nitpicking - Think name, description, resourceGroupName, labels, scraping, azureMetricConfiguration would make a more logical ordering

Reply:
Sure - easy to sort.

Reply:
Thanks!